public/locales/en/translation.json

```diff
@@ -69,6 +69,10 @@
             "launchAborted": "Launch aborted",
             "moving": "Error Moving Game {{error}}",
             "title": "Error",
+            "ubisoft-connect": {
```

Comment:
This is not related but was created when I ran `yarn i18n` Probably some leftover from some previous PR.